packages/camera/camera_avfoundation/example/ios/RunnerTests/SampleBufferTests.swift

```diff
@@ -135,7 +132,7 @@ final class CameraSampleBufferTests: XCTestCase {
     camera.resumeVideoRecording()
 
     camera.captureOutput(
-      camera.captureVideoOutput.avOutput, didOutputSampleBuffer: sampleBuffer, from: connectionMock)
+      camera.captureVideoOutput.avOutput, didOutput: sampleBuffer)
```

Comment:
why is connection param removed? was it not used?

Reply:
It wasn't used it was included in the signature to match the `AVCaptureVideoDataOutputSampleBufferDelegate` protocol. I've split the method into one matching the protocol and one matching our actual needs but I can make it work without splitting